src/controllers/productController/productController.ts

```diff
@@ -111,7 +110,7 @@ async updateProduct(req: Request, res: Response): Promise<void> {
     const productId: number = Number(req.params.productId);
     try {
       const userId = req.body.userId;
-      const user = await UserService.getUserById(userId) as any;
+      const user = await User.findByPk(userId) as any;
```

Comment:
Having many methods like findbypk or findbyseller is not a viable approach, try to us rind by fields which can find by mentioned fileds